webui/react/config-overrides.js

```diff
@@ -14,9 +14,23 @@ const path = require('path');
 const webpack = require('webpack');
 const jestConfig = require('./jest.config');
 
-const IS_DEV = process.env.NODE_ENV === 'development';
+const IS_DEV = process.env.DET_NODE_ENV === 'development';
```

Comment:
**question:** what's the motivation for this change? I might be missing something but I don't see any other references of `DET_NODE_ENV` else where. A little worried that because `NODE_ENV` is so heavily used and common in node and possibly CRA, this might have unforeseen consequences.

Reply:
Ah I think I get what you're doing. If the user sets this env var then during both `npm start` and `npm run build` will skip optimizations for a faster build?

Reply:
**suggestion:** I think this is totally worth adding to our `/webui/react/README.md`

Reply:
the main issue I was addressing is that CRA and what we use to override does not allow us to externally change NODE_ENV so that IS_DEV that was defined there could never become true and easiest workaround was just to use another env var

Reply:
idea being you set this to always be development in your environment and only unset it if you need to

Reply:
> suggestion: I think this is totally worth adding to our /webui/react/README.md

agreed. not being able to use NODE_ENV makes it a bit awkward  IMO

Reply:
I added a env var section to our readme